### PR TITLE
blog: real programmers log('here')

### PR DESCRIPTION
**Pull request by nicks**
Hello @ellenkorbes,

Please review the following commits I made in branch nicks/pls:

42d6aad4b770d971b82654f448795e3934abdf6b (2021-05-21 15:17:16 -0400)
blog: real programmers log('here')

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: blog/_posts/2021-05-21-real-programmers-log.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+right logs, and bring it back up.
+
+The big advantage of a declarative API for streaming logs is that we can
+dynamically change the logs that are currently streaming.
+
+- `tilt delete` can [delete](https://docs.tilt.dev/cli/tilt_delete.html) a PodLogStream so that the Tilt environment stops
+  streaming the logs.
+
+- `tilt apply` can create a new PodLogStream or modify an existing one, so that
+  you can add new containers, or ignore annying ones. Usually you
```
Comment (ellenkorbes) [inline blog/_posts/2021-05-21-real-programmers-log.md:182]:
```suggestion
  you can add new containers, or ignore annoying ones. Usually you
```

File: blog/_posts/2021-05-21-real-programmers-log.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+  write the PodLogStream config in YAML in a file, then feed it to [apply](https://docs.tilt.dev/cli/tilt_apply.html)
```
Comment (ellenkorbes) [inline blog/_posts/2021-05-21-real-programmers-log.md:183]:
```suggestion
  write the PodLogStream config in YAML in a file, then feed it to [apply](https://docs.tilt.dev/cli/tilt_apply.html).
```

**Comment by ellenkorbes** [comment]
Would it be a lot of work to add short examples to the last two bullet points? By that point in the text I really wanna see them

**Review by ellenkorbes — APPROVED** [review]

File: blog/_posts/2021-05-21-real-programmers-log.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+## Wrapping Up
+
+I hope this post gave you a sense of how we're thinking about logs in
+multi-service dev, other tools you can reach for to help you explore them, and
+some of the APIs we're working on in Tilt to make them easier to grok.
+
+[^1]: I'm approaching logs with the axiom that logs for development 
+      (where you're talking to yourself) 
+      are a different style of communication than logs for production 
+      (where you're talking to other people.)
```
Comment (lizzthabet) [inline blog/_posts/2021-05-21-real-programmers-log.md:204]:
```suggestion
      (where you're talking to other people). 
```
A tiny grammar note!

File: blog/_posts/2021-05-21-real-programmers-log.md (hunk 1 above)
Comment (nicks) [reply blog/_posts/2021-05-21-real-programmers-log.md]: done

File: blog/_posts/2021-05-21-real-programmers-log.md (hunk 3 above)
Comment (nicks) [reply blog/_posts/2021-05-21-real-programmers-log.md]: done

**Comment by nicks** [comment]
@ellenkorbes i added some examples! I was worried that I had worn out my welcome and was trying to wrap it up

**Pull request merged**